### ISSUE TITLE
Prefer "Any" to "Count() > 0"

### ISSUE THREAD

**Issue by giggio** [code-cracker/code-cracker#490]
If you find a call to `Count() > 0` and the object supports `Any()`, change it `Any()`.
If you find a call to `Count(predicate) > 0` and the object supports `Any(<predicate type>)`, change it `Any(predicate)`.

So this:

````csharp
var any = myEnumerable.Count() > 0;
````

Becomes:

````csharp
var any = myEnumerable.Any();
````

And:

````csharp
var any = myEnumerable.Count(i => i > 3) > 0;
````

Becomes:

````csharp
var any = myEnumerable.Any(i => i > 3);
````

Diagnostic id: `CC0099`
Severity: Information
Category: Style

**Comment by akamud** [comment]
I would like to work on this

**Comment by giggio** [comment]
Go for it!

**Comment by giggio** [comment]
@akamud Remember to branch from vnext, and to pull into it. I have added you to the collaborators team, so I can assign the issue to you.

**Comment by akamud** [comment]
Will do.

**Comment by giggio** [comment]
Closed on #528
